docs/GITFLOW.md

```diff
@@ -0,0 +1,39 @@
+## Git(hub) Flow:
+
+1. Checkout and pull latest `dev` branch
+2. Make your new feature branch following naming conventions
+3. Do your changes, and make pull request in github back to `dev` branch
+4. Ask for review, or(and) assign people on your pr to speed up process
+5. After receiving enough approves and testing, merge your pull request to `dev`
```

Comment:
In case you don't get all the approves quickly, should you take care of keeping your branch up-to-date with deve?

Reply:
until branch is merged, work is not done, and feature branch should be consistently updated

that also works as additional motivational source to merge features quickly, rather than making lots of PR's